src/test/scala/com/yotpo/metorikku/metric/stepActions/dataQuality/HasUniquenessTest.scala

```diff
@@ -0,0 +1,70 @@
+package com.yotpo.metorikku.metric.stepActions.dataQuality
+
+import com.amazon.deequ.checks.Check
+import com.yotpo.metorikku.metric.stepActions.dataQuality.operators.HasUniqueness
+import org.apache.log4j.{Level, Logger}
+import org.apache.spark.sql.SparkSession
+import org.scalatest.{BeforeAndAfterEach, FunSuite}
+
+class HasUniquenessTest extends FunSuite with BeforeAndAfterEach {
+  private var sparkSession : SparkSession = _
+  Logger.getLogger("org").setLevel(Level.WARN)
+
+  override def beforeEach() {
+    sparkSession = SparkSession.builder().appName("dq tests")
+      .master("local")
+      .config("", "")
+      .getOrCreate()
+  }
+
+  private def valideHasUniquenessOverDf(employeeData: Seq[(String, Int, String, Int, Int)], columns: Seq[String],
+                                        fraction: Double, operator: String, level: String) = {
+    val sqlContext = sparkSession.sqlContext
+    val hasUniquenessCheck = new HasUniqueness(level = Some(level),columns = columns, fraction, operator)
+    val dqCheckDefinitionList = DataQualityCheckList(List[DataQualityCheck](DataQualityCheck(None, None, hasUniqueness = Some(hasUniquenessCheck))), None)
+    import sqlContext.implicits._
+
+    val dfName = "employee_data"
+    val df = employeeData.toDF(dfName, "id", "name", "fake", "fake2")
```

Comment:
first column is the employee_name isn't it?
Not that it matters but it confused me...